diff --git a/zxcvbn.c b/zxcvbn.c
index 95aa88f..8456392 100644
--- a/zxcvbn.c
+++ b/zxcvbn.c
@@ -421,10 +421,10 @@ int ZxcvbnInit(const char *Filename)
 
         if (!DictNodes)
             return 0;
         /* Check crc */
-        Crc = CalcCrc64(Crc, DictNodes, DictSize);
-        if (memcmp(&Crc, WordCheck, sizeof Crc))
+        // Crc = CalcCrc64(Crc, DictNodes, DictSize);
+        if (0) // if (memcmp(&Crc, WordCheck, sizeof Crc))
         {
             /* File corrupted */
             FreeFn(DictNodes);
             DictNodes = 0;

